Project-3/README.md

```diff
@@ -11,67 +11,466 @@ For any question involving the use of Protege, please be sure to import:
+  (In the following example I sometimes use terms like "is" between, e.g. "Sally" and 
+  "instance_of. Technically speaking, there is no such a thing (see BS book, pag. 117))
+
+(a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+
+Sally instance_of object
+Tuesday instance_of one-dimensional temporal region 
+Wednesday instance of one-dimensional temporal region 
+Arm instance_of fiat object part
+“Sally participates in having at least one arm on Tuesday” instance of occurrent 
+“Sally participates in having no arms on Wednesday” instance of occurrent
+“Sally participates in having at most one arm on Wednesday” instance of occurrent
+
+Due to the vagueness of the statement above, we have narrowed the possible answers to the following two cases:
+
+“"Sally participates in having at least one arm on Tuesday" precedes "Sally participates
+in having no arms on Wednesday"”
+“"Sally participates in having at least one arm on Tuesday" precedes "Sally participates 
+in having at most one arm on Wednesday"”
+
   (b) Every liver has some cell as part at all times it exists.
+
+  We already have cell in the ontology.
+  Liver is a class in Uberon (UBERON_0002107), which is a subclass of material entity. 
+  https://ontobee.org/ontology/UBERON?iri=http://purl.obolibrary.org/obo/UBERON_0002107
+
+  Argument ex auctoritate from BS for the object property has_part_at_all_times:
+  https://youtu.be/fkkWkTIxrNQ
+
+  Liver has_part_at_all_times Cell
+
   (c) John was a child, then an adult, then a senior. 
+
+  John instance_of object
+  childhood instance_of occurrent.
+  adulthood instance_of occurrent.
+  seniorhood instance_of occurrent.
+
+  “John participates in childhood precedes John participates in adulthood 
+  which precedes John participates in seniorhood.”
+
   (d) Goofus and Gallant are married at each point in a three year span. 
 ```
+  Goofus instance_of object
+  Gallant instance_of object
+  Marriage instance_of occurent
+  “Three years span 1” instance_of one-dimensional temporal region.
+  Notice that the original phrase doesn’t say anything about Goofus and Gallant being
+  married to each other, and we won’t represent such a fact.
 
-4. Using the language of First-Order Logic, represent the following natural language expressions; you are welcome to introduce new terms where needed: 
+  If zero-dimensional temporal region t1 part_of one-dimensional temporal region 
+  “three years span 1”, then Goofus participates in marriage at t1 and Galland participates
+  in marriage at t1.
+
+4. Using the language of First-Order Logic, represent the following natural language expressions;
+you are welcome to introduce new terms where needed: 
 ```
   (a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+
+  ∃x (Tx ∧ ∃y (Hsy∧Ay)) ∧ ∃x (Wx ∧ ~∃y(Hsy∧Ay))
+  T: Tuesday
+  H: has
+  A: arm
+  W: Wednesday
+  s: Sally
+
   (b) Every liver has some cell as part at all times it exists.
+
+  ∀x∃y(Lx→Cy∧Pyx)
```

Comment:
This permits models in which there are no livers, i.e. trivially true. Exclude those models.